packages/api/src/types.ts

```diff
@@ -32,6 +32,10 @@ export interface ApiOptions extends Pick<ApiOptionsBase, Exclude<keyof ApiOption
    * default 10000 ms, 0 indicates no limit
    */
   timeout?: number;
+
```

Comment:
can you please add some doc comments here so developers get the nice hints when using this package